tensorflow/python/ops/image_ops_impl.py

```diff
@@ -4044,6 +4044,28 @@ def draw_bounding_boxes_v2(images, boxes, colors, name=None):
 
   Returns:
     A `Tensor`. Has the same type as `images`.
+
+  Usage Example:
```

Comment:
Add an empty line after `Usage Example:` to make the doctest show properly on the documentation website.

Reply:
I noticed the test fails after the first empty line in the doctest output. Does adding `<BLANKLINE>` in the empty lines fix that? 
```python
  <tf.Tensor: shape=(1, 3, 3, 3), dtype=float32, numpy=
  array([[[[1., 0., 0.],
          [1., 0., 0.],
          [1., 0., 0.]],

          [[1., 0., 0.],
          [0., 0., 0.],
          [1., 0., 0.]],

          [[1., 0., 0.],
          [1., 0., 0.],
          [1., 0., 0.]]]], dtype=float32)>
```

Reply:
I think you don't need the empty lines in the output. At least I know that removing the empty lines fixed other PRs

Reply:
OK, I will remove those empty lines. Thanks!